lib/msal-browser/src/broker/nativeBroker/IPlatformAuthHandler.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { PlatformBrokerRequest } from "./PlatformBrokerRequest.js";
+import { PlatformBrokerResponse } from "./PlatformBrokerResponse.js";
+
+/**
+ * Interface for the Platform Broker Handlers
+ */
+export interface IPlatformAuthHandler {
+    getExtensionId(): string | undefined;
+    getExtensionVersion(): string | undefined;
+    sendMessage(
+        request: PlatformBrokerRequest
+    ): Promise<PlatformBrokerResponse>;
+    validatePlatformBrokerResponse(response: object): PlatformBrokerResponse;
```

Comment:
Is this called anywhere outside of `sendMessage`? Probably doesn't need to be part of the interface/public